styleguide/markdown-syntax-style.md

```diff
@@ -444,7 +378,7 @@ Line 3, column 1  | Line 3, column 2
 
 #### Responsive tables
```

Comment:
The information about callouts and tables feels like it might be in the wrong section to me. I know it's technically something custom on top of Markdown, but I'm not sure if that distinction is important to our authors. I expect they're just looking how to format something to work in our site. 🤔

Reply:
Done.